docs/zh/UserGuide/API/InfluxDB-Protocol.md

```diff
@@ -24,26 +24,26 @@
 假如您原先接入 InfluxDB 的业务代码如下：
 
 ```java
-InfluxDB influxDB = InfluxDBFactory.connect(openurl, username, password);
+InfluxDB influxDB=InfluxDBFactory.connect(openurl,username,password);
 ```
 
 您只需要将 InfluxDBFactory 替换为 **IoTDBInfluxDBFactory** 即可实现业务向 IoTDB 的切换：
 
 ```java
-InfluxDB influxDB = IoTDBInfluxDBFactory.connect(openurl, username, password);
+InfluxDB influxDB=IoTDBInfluxDBFactory.connect(openurl,username,password);
 ```
```

Comment:
Thanks for your review, in this file, I use the formatting shortcut provided by IDEA. Maybe it automatically formats the Java code block of makedown into this style. I think this may not be elegant. I have reverted to the previous version.